app/src/main/java/com/monstar_lab/sample/android/kotlin/MainActivity.kt

```diff
@@ -36,4 +36,7 @@ class MainActivity : AppCompatActivity() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    public fun aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/sample-android-kotlin/blob/a2e29435ca6151fbf188298f8b320943d3aa73a6/app/src/main/java/com/monstar_lab/sample/android/kotlin/MainActivity.kt#L40"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>